lib/streamlit/commands/execution_control.py

```diff
@@ -43,6 +44,7 @@ def stop() -> NoReturn:
     raise StopException()
 
 
+@gather_metrics("rerun")
```

Comment:
nit: Are we removing the `experimental_` with the same release? If not, it might be better to use `experimental_rerun` here as metric name until it's de-experimentalized.